serializer-deserializer/src/main/java/com/amazonaws/services/schemaregistry/utils/apicurio/FileDescriptorUtils.java

```diff
@@ -90,48 +90,48 @@ public class FileDescriptorUtils {
     // rpc options
     private static final String IDEMPOTENCY_LEVEL_OPTION = "idempotency_level";
 
-    private static final OptionElement.Kind booleanKind =  OptionElement.Kind.BOOLEAN;
-    private static final OptionElement.Kind stringKind =  OptionElement.Kind.STRING;
-    private static final OptionElement.Kind enumKind =  OptionElement.Kind.ENUM;
+    private static final OptionElement.Kind booleanKind = OptionElement.Kind.BOOLEAN;
+    private static final OptionElement.Kind stringKind = OptionElement.Kind.STRING;
+    private static final OptionElement.Kind enumKind = OptionElement.Kind.ENUM;
 
     public static FileDescriptor[] baseDependencies() {
         //Support all the Protobuf WellKnownTypes
         //and the protos from Google API, https://github.com/googleapis/googleapis
-        return new FileDescriptor[] {
-            ApiProto.getDescriptor().getFile(),
-            FieldMaskProto.getDescriptor().getFile(),
-            SourceContextProto.getDescriptor().getFile(),
-            StructProto.getDescriptor().getFile(),
-            TypeProto.getDescriptor().getFile(),
-            TimestampProto.getDescriptor().getFile(),
-            WrappersProto.getDescriptor().getFile(),
-            AnyProto.getDescriptor().getFile(),
-            EmptyProto.getDescriptor().getFile(),
-            DurationProto.getDescriptor().getFile(),
-            TimeOfDayProto.getDescriptor().getFile(),
-            DateProto.getDescriptor().getFile(),
-            CalendarPeriodProto.getDescriptor().getFile(),
-            ColorProto.getDescriptor().getFile(),
-            DayOfWeek.getDescriptor().getFile(),
-            LatLng.getDescriptor().getFile(),
-            FractionProto.getDescriptor().getFile(),
-            MoneyProto.getDescriptor().getFile(),
-            MonthProto.getDescriptor().getFile(),
-            PhoneNumberProto.getDescriptor().getFile(),
-            PostalAddressProto.getDescriptor().getFile(),
-            CalendarPeriodProto.getDescriptor().getFile(),
-            LocalizedTextProto.getDescriptor().getFile(),
-            IntervalProto.getDescriptor().getFile(),
-            ExprProto.getDescriptor().getFile(),
-            QuaternionProto.getDescriptor().getFile(),
-            PostalAddressProto.getDescriptor().getFile(),
-            ProtobufSchemaMetadata.getDescriptor().getFile(),
-            Decimals.getDescriptor().getFile()
+        return new FileDescriptor[]{
+                ApiProto.getDescriptor().getFile(),
```

Comment:
Can you revert these formatting change since it increases the difficultly of code review?

Reply:
Do you have any formatting tools in place to help me?

Reply:
Otherwise adding `&w=1` in the review URL will only show diff without space. https://github.blog/2018-05-01-ignore-white-space-in-code-review/

Reply:
I have just remove the whitespace anyway